### PR TITLE
Uprev to rubocop 0.35.1

### PR DESCRIPTION
**Pull request by robert-w-gries**
Rubocop now clean for version 0.35.1.  Minitest has errors, but they happen for both `feature/rubocop-uprev` and `develop` branches.

### CODE REVIEW

File: lib/cisco_node_utils/bgp.rb [hunk 1]
```diff
@@ -56,8 +56,10 @@ def self.routers
 
       hash_final = {}
       asnum = asnum.to_i unless /\d+.\d+/.match(asnum)
+      # rubocop:disable Style/AlignHash
       hash_tmp = { asnum =>
         { 'default' => RouterBgp.new(asnum, 'default', false) } }
+      # rubocop:enable Style/AlignHash
```
Comment (RichWellum) [inline lib/cisco_node_utils/bgp.rb]: Why not just fix this one?

Comment (robert-w-gries) [inline lib/cisco_node_utils/bgp.rb]: Rubocop's autocorrect is:

```
hash_tmp = { asnum =>
                       { 'default' => RouterBgp.new(asnum, 'default', false) } }
```

And I couldn't find a cleaner solution that Rubocop approved of.

Comment (RichWellum) [inline lib/cisco_node_utils/bgp.rb]: How about:

hash_tmp = 
  { asnum =>
      { 'default' => RouterBgp.new(asnum, 'default', false) } }

or:

hash_tmp = { asnum => { 'default' => RouterBgp.new(asnum, 'default', false) } }

Comment (chrisvanheuveln) [inline lib/cisco_node_utils/bgp.rb]: I gave you guys this one once before :-)

```
hash_tmp = {
  asnum => { 'default' => RouterBgp.new(asnum, 'default', false) }
}
```

File: lib/cisco_node_utils/router_ospf_vrf.rb [hunk 2]
```diff
@@ -56,8 +56,10 @@ def self.vrfs
       RouterOspf.routers.each do |instance|
         name = instance[0]
         vrf_ids = config_get('ospf', 'vrf', name: name)
+        # rubocop:disable Style/AlignHash
         hash_tmp = { name =>
           { 'default' => RouterOspfVrf.new(name, 'default', false) } }
+        # rubocop:enable Style/AlignHash
```
Comment (chrisvanheuveln) [inline lib/cisco_node_utils/router_ospf_vrf.rb]: ditto

**Comment by robert-w-gries** [comment]
Made @chrisvanheuveln's suggested change

**Pull request merged**